2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Configure stats pid to 28312
2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Loading settings from C:\Users\OJH\.config\wandb\settings
2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Loading settings from c:\Users\OJH\OneDrive\SKKU\연구자료\Catalyst AI_rev\New Code\wandb\settings
2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-04 17:16:18,795 INFO    MainThread:28312 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-04 17:16:18,796 INFO    MainThread:28312 [wandb_init.py:_log_setup():526] Logging user logs to c:\Users\OJH\OneDrive\SKKU\연구자료\Catalyst AI_rev\New Code\wandb\run-20240304_171618-bkpgojyn\logs\debug.log
2024-03-04 17:16:18,796 INFO    MainThread:28312 [wandb_init.py:_log_setup():527] Logging internal logs to c:\Users\OJH\OneDrive\SKKU\연구자료\Catalyst AI_rev\New Code\wandb\run-20240304_171618-bkpgojyn\logs\debug-internal.log
2024-03-04 17:16:18,796 INFO    MainThread:28312 [wandb_init.py:init():566] calling init triggers
2024-03-04 17:16:18,796 INFO    MainThread:28312 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-04 17:16:18,796 INFO    MainThread:28312 [wandb_init.py:init():591] re-initializing run, found existing run on stack: d481dl8r
2024-03-04 17:16:18,797 INFO    MainThread:28312 [wandb_run.py:_finish():1970] finishing run jonghwan_oh/ipse_skku/d481dl8r
2024-03-04 17:16:18,797 INFO    MainThread:28312 [jupyter.py:save_history():445] not saving jupyter history
2024-03-04 17:16:18,797 INFO    MainThread:28312 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-04 17:16:18,797 INFO    MainThread:28312 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-04 17:16:18,797 INFO    MainThread:28312 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-04 17:16:18,798 INFO    MainThread:28312 [wandb_run.py:_restore():2193] restore
2024-03-04 17:16:18,798 INFO    MainThread:28312 [wandb_run.py:_restore():2199] restore done
2024-03-04 17:16:24,441 INFO    MainThread:28312 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-04 17:16:24,441 INFO    MainThread:28312 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-04 17:16:24,443 INFO    MainThread:28312 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-04 17:16:24,746 INFO    MainThread:28312 [wandb_init.py:init():616] starting backend
2024-03-04 17:16:24,746 INFO    MainThread:28312 [wandb_init.py:init():620] setting up manager
2024-03-04 17:16:24,749 INFO    MainThread:28312 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-04 17:16:24,750 INFO    MainThread:28312 [wandb_init.py:init():628] backend started and connected
2024-03-04 17:16:24,760 INFO    MainThread:28312 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-04 17:16:24,760 INFO    MainThread:28312 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-03-04 17:16:24,760 INFO    MainThread:28312 [wandb_init.py:init():720] updated telemetry
2024-03-04 17:16:24,801 INFO    MainThread:28312 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-04 17:16:26,139 INFO    MainThread:28312 [wandb_run.py:_on_init():2262] communicating current version
2024-03-04 17:16:26,428 INFO    MainThread:28312 [wandb_run.py:_on_init():2271] got version response 
2024-03-04 17:16:26,428 INFO    MainThread:28312 [wandb_init.py:init():804] starting run threads in backend
2024-03-04 17:16:32,549 INFO    MainThread:28312 [wandb_run.py:_console_start():2241] atexit reg
2024-03-04 17:16:32,549 INFO    MainThread:28312 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-04 17:16:32,549 INFO    MainThread:28312 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-04 17:16:32,549 INFO    MainThread:28312 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-04 17:16:32,550 INFO    MainThread:28312 [wandb_init.py:init():847] run started, returning control to user process
